homeassistant/components/netatmo/climate.py

```diff
@@ -10,35 +10,43 @@
 from homeassistant.components.climate import ClimateDevice, PLATFORM_SCHEMA
 from homeassistant.components.climate.const import (
     HVAC_MODE_AUTO, HVAC_MODE_HEAT, HVAC_MODE_OFF,
-    PRESET_AWAY,
+    PRESET_AWAY, PRESET_BOOST,
     CURRENT_HVAC_HEAT, CURRENT_HVAC_IDLE,
-    SUPPORT_TARGET_TEMPERATURE, SUPPORT_PRESET_MODE
+    SUPPORT_TARGET_TEMPERATURE, SUPPORT_PRESET_MODE,
+    DEFAULT_MIN_TEMP
 )
 from homeassistant.const import (
-    TEMP_CELSIUS, ATTR_TEMPERATURE, CONF_NAME, PRECISION_HALVES)
+    TEMP_CELSIUS, ATTR_TEMPERATURE, CONF_NAME, PRECISION_HALVES, STATE_OFF)
 from homeassistant.util import Throttle
 
 from .const import DATA_NETATMO_AUTH
 
 _LOGGER = logging.getLogger(__name__)
 
-PRESET_FROST_GUARD = 'frost_guard'
-PRESET_MAX = 'max'
+PRESET_FROST_GUARD = 'frost guard'
 PRESET_SCHEDULE = 'schedule'
 
 SUPPORT_FLAGS = (SUPPORT_TARGET_TEMPERATURE | SUPPORT_PRESET_MODE)
 SUPPORT_HVAC = [HVAC_MODE_HEAT, HVAC_MODE_AUTO, HVAC_MODE_OFF]
 SUPPORT_PRESET = [
-    PRESET_AWAY, PRESET_FROST_GUARD, PRESET_SCHEDULE, PRESET_MAX,
+    PRESET_AWAY, PRESET_BOOST, PRESET_FROST_GUARD, PRESET_SCHEDULE,
 ]
 
-STATE_NETATMO_SCHEDULE = 'schedule'
+STATE_NETATMO_SCHEDULE = PRESET_SCHEDULE
 STATE_NETATMO_HG = 'hg'
-STATE_NETATMO_MAX = PRESET_MAX
+STATE_NETATMO_MAX = 'max'
 STATE_NETATMO_AWAY = PRESET_AWAY
-STATE_NETATMO_OFF = "off"
+STATE_NETATMO_OFF = STATE_OFF
 STATE_NETATMO_MANUAL = 'manual'
 
+PRESET_MAP_NETATMO = {
+    PRESET_FROST_GUARD: STATE_NETATMO_HG,
+    PRESET_BOOST: STATE_NETATMO_MAX,
```

Comment:
This is a breaking change, please include a paragraph for the notes.

Reply:
Why is it breaking? This has not been released yet.